.gitignore

```diff
@@ -38,3 +38,4 @@ oprofile*
 position.txt
 *.9
 *.glade.h
+/build/
```

Comment:
The .gitignore file commited to git reflects only files that are created during the build process.  This path /build/ is not.  You can add your own ignores in $GIT_DIR/info/exclude -- see man gitignore(5) for more information.